python_scripts/trees.py

```diff
@@ -320,20 +321,20 @@ def plot_decision_function(X, y, clf, ax=None):
 
 
 # %% [markdown]
-# As we visually assess, the partition defined by < 42 mm has mainly Adelie
-# penguin and only 2 samples which we could considered misclassified. However,
-# on the partition >= 42 mm, we cannot differentiate Gentoo and Chinstrap
-# (while they are almost twice more Gentoo).
+# As we visually assessed, the partition defined by < 42 mm has mainly Adelie
+# penguin and only 2 samples that are misclassified. However,
+# in the partition >= 42 mm, we cannot differentiate well between Gentoo and
+# Chinstrap (while they are almost twice more Gentoo).
 #
-# We should come with a statistical measure which combine the class
-# probabilities together that can be used as a criterion to qualify the purity
+# We should use a statistical measure that uses the all class
+# probabilities, as the criterion to qualify the purity
 # of a partition. We will choose as an example the entropy criterion (also used
 # in scikit-learn) which is one of the possible classification criterion.
 #
 # The entropy is defined as: $H(X) = - \sum_{k=1}^{K} p(X_k) \log p(X_k)$
```

Comment:
I think it would be worth defining what the symbols represent.

Reply:
+1

Reply:
v